src/components/auth/login/login.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useContext, useEffect } from 'react'
+import './login.css'
+import * as AuthApis from '../../../api/authApi'
+import { Link, useNavigate } from 'react-router-dom'
+import { useAuth } from '../../../contexts/globalContext'
+import { useGlobal } from '../../../contexts/globalContext'
```

Comment:
you can use [Absolute Imports](https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8). Imports will look clean